BinanceExchange.API/APIProcessor.cs

```diff
@@ -82,7 +82,7 @@ public void SetCacheTime(TimeSpan time)
             if (errorObject == null) throw new BinanceException("Unexpected Error whilst handling the response", null);
             errorObject.RequestMessage = requestMessage;
             var exception = CreateBinanceException(message.StatusCode, errorObject);
-            _logger.Error($"Error Message Received:", exception);
+            _logger.Error($"Error Message Received:{errorJson}", exception);
```

Comment:
I might be wrong, but isn't the message added to the `BinanceException`? If not and users want the whole `errorJson` this could be a good addition

Reply:
I can only see a message like 'One or more exceptions occurred etc.' from the log entry of exception and there wasn't anymore information about the inner exception. So I think this will be a useful fix for wrapper.